services/catalog/src/simcore_service_catalog/core/events.py

```diff
@@ -32,6 +39,9 @@ async def stop_app() -> None:
             logger.info("Application stopping")
             if app.state.settings.postgres.enabled:
                 await close_db_connection(app)
+            if app.state.settings.director.enabled:
```

Comment:
Same as above

I could not select the entire function, but my question is why are you wrapping the handler in  a function?

Reply:
same as above